src/TestRotationMatrix.java

```diff
@@ -0,0 +1,148 @@
+//: object/TestRotationMatrix.java
+
+import junit.framework.*;
+
+import java.lang.*;
+
+/**
+ * JUnit Class to test the methods from the RotationMatrix Class.
+ *
+ * @author Julian Mateu
+ * @author julianmateu@gmail.com
+ * @version 1.0
+ */
+public class TestRotationMatrix extends ProjectTest {
+
+    private static final int SIZE = 3;
+
+    /**
+     * Constructor
+     */
+    public TestRotationMatrix(String name) {
+        super(name);
+    }
+
+    /**
+     * Tester for the equals() method.
+     */
```

Comment:
IMO there is no need for this sort of comment if your test is called `testEquals`.